### DISCUSSION TITLE
Version 3.1.0

### DISCUSSION THREAD

**Discussion by Tatsu-syo** [Tatsu-syo/noMeiryoUI#55] [General]
- Windows 11 2022 Update(22H2)ではタイトルバーに透明を設定したときの問題を回避するため、タイトルバーのフォントを強制的に標準のフォント設定にするようにしました。
- 同様の理由でWindows 11 2022 Update(22H2)ではタイトルバーのフォントを選択できなくしました。
- Windows 11 2022 UpdateではWindows 11 2022 Updateと表示するようにしました。

----

- On Windows 11 2022 Update(22H2) force set system setting to title bar font, to avoid problem when transparent effect is applied on title bar,
- By same reason, title bar font select button is disabled on Windows 11 2022 Update(22H2).
- Show version number &quot;Windows 11 2022 Update&quot; on Windows 11 2022 Update,

<hr /><em>This discussion was created from the release <a href='https://github.com/Tatsu-syo/noMeiryoUI/releases/tag/TAG-3.1.0'>Version 3.1.0</a>.</em>

**Comment by Tatsu-syo** [comment]
This version is made for Windows 11 22H2(2022 release).
To avoid problem, set title bar font is disabled on Windows 11 22H2.
To release rapidly, this version includes only Japanese and English language files.
If you can help translate your language, please contact me or fork and pull request or you can.

This discussion how to treat Windows 11 22H2 and functions discussions are welcome.

**Reply by overartem** [reply]
after selecting my custom font, the app froze and closed. The same with Version 3.1.1  (Windows11 PRo 22H2 - 22621.1702)
